[squeezebox] NumberFormatException when starting new song

I had the binding in debug mode and saw this. Somewhere along the line it looks like `repeating_stream` is no longer an int. This value isn't used anywhere (and I don't even know what it is but it always seems to have the same value as duration). It's not causing a problem other than being unable to retrieve the value of custom buttons (e.g. like and unlike song). I'll submit a PR to just remove it from the DTO.

```
2026-07-30 13:20:27.289 [DEBUG] [queezebox.internal.handler.SqueezeBoxServerHandler] - JsonSyntaxException parsing status response: {"method":"slim.request","result":**snip**,"repeating_stream":202.085083333333},"id":1,"params":["54:78:c9:6f:a1:86",["status","-","tags:yagJlNKjcB"]]}
com.google.gson.JsonSyntaxException: java.lang.NumberFormatException: Expected an int but was 202.085083333333 at line 1 column 644 path $.result.repeating_stream
        at com.google.gson.internal.bind.TypeAdapters$7.read(TypeAdapters.java:267)
        at com.google.gson.internal.bind.TypeAdapters$7.read(TypeAdapters.java:257)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$2.readIntoField(ReflectiveTypeAdapterFactory.java:271)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$FieldReflectionAdapter.readField(ReflectiveTypeAdapterFactory.java:559)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:517)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$2.readIntoField(ReflectiveTypeAdapterFactory.java:271)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$FieldReflectionAdapter.readField(ReflectiveTypeAdapterFactory.java:559)
        at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:517)
        at com.google.gson.Gson.fromJson(Gson.java:1358)
        at com.google.gson.Gson.fromJson(Gson.java:1259)
        at com.google.gson.Gson.fromJson(Gson.java:1169)
        at com.google.gson.Gson.fromJson(Gson.java:1106)
        at org.openhab.binding.squeezebox.internal.handler.SqueezeBoxServerHandler$SqueezeServerListener.updateCustomButtons(SqueezeBoxServerHandler.java:1020)
        at org.openhab.binding.squeezebox.internal.handler.SqueezeBoxServerHandler$SqueezeServerListener.lambda$18(SqueezeBoxServerHandler.java:879)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
        at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
        at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: java.lang.NumberFormatException: Expected an int but was 202.085083333333 at line 1 column 644 path $.result.repeating_stream
        at com.google.gson.stream.JsonReader.nextInt(JsonReader.java:1353)
        at com.google.gson.internal.bind.TypeAdapters$7.read(TypeAdapters.java:265)
        ... 19 more
```

--- a/bundles/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/dto/StatusResultDTO.java
+++ b/bundles/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/dto/StatusResultDTO.java
@@ -79,9 +79,6 @@ public class StatusResultDTO {
     @SerializedName("remote")
     public String remote;
 
-    @SerializedName("repeating_stream")
-    public Integer repeatingStream;
-
     @SerializedName("seq_no")
     public Integer sequenceNumber;
 

